usaspending_api/common/etl/spark.py

```diff
@@ -31,10 +31,13 @@
     PopCongressionalDistrict,
     DisasterEmergencyFundCode,
 )
+from usaspending_api.recipient.delta_models import (
+    sam_recipient,
+)
 from usaspending_api.submissions.models import SubmissionAttributes, DABSSubmissionWindowSchedule
 
 MAX_PARTITIONS = CONFIG.SPARK_MAX_PARTITIONS
-_RDS_REF_TABLES = [
+usas_RDS_REF_TABLES = [
```

Comment:
Constants have a convention of all upper case names. Please change to `USAS_RDS_REF_TABLES`.

Reply:
Renamed and updated.